packages/core/src/domain/configuration/configuration.ts

```diff
@@ -68,8 +68,8 @@ export interface Configuration extends TransportConfiguration {
   // Batch configuration
   batchBytesLimit: number
   flushTimeout: number
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
What about renaming `flushTimeout` to something like `batchDurationLimit`?

Reply:
Fine by me :)